docs/advanced-features/i18n-routing.md

```diff
@@ -0,0 +1,134 @@
+# Internationalized Routing
+
+<details>
+  <summary><b>Examples</b></summary>
+  <ul>
+    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/i18n-routing">i18n routing</a></li>
+  </ul>
+</details>
+
+Next.js has built in support for i18n routing since v9.5.7. With the built-in support you can provide a list of locales you want to support, the default locale, and domain specific locales and Next.js will help handle the routing for these values.
```

Comment:
```suggestion
Next.js has built-in support for internationalized (i18n) routing since `v9.5.7`. You can provide a list of locales, the default locale, and domain-specific locales and Next.js will automatically handle the routing.
```

Reply:
Do we include an external link for why people abbreviate this to i18n on the (i18n) part? I definitely had to look it up the first time.

Reply:
Added an external link to the wikipedia page explaining the naming